assessment/catscores.php

```diff
@@ -88,7 +88,7 @@ function printoutcomes($arr,$ind,&$outcomenames, &$catscore, &$catposs) {
 			continue;
 		} elseif (0==strncmp($category,"AID-",4)) { //category is another assessment
 			$categoryaid=substr($category,4);

```

Comment:
Will you please make one more small change.  Change this line to
`$categoryaid= intval(substr($category,4));`

I think someone could try to inject SQL here by creating a custom category like `AID-4'; DROP TABLE imas_courses; --`.  Rather than escape $categoryaid, since we know the categoryaid is supposed to be an integer we can use intval to prevent this.